keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigYamlParsingFilesTests.kt

```diff
@@ -0,0 +1,122 @@
+/*
+ *
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.spinnaker.keel.rest
+
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import com.netflix.spinnaker.keel.api.ec2.ApplicationLoadBalancerSpec
+import com.netflix.spinnaker.keel.api.ec2.ClassicLoadBalancerSpec
+import com.netflix.spinnaker.keel.api.ec2.ClusterSpec
+import com.netflix.spinnaker.keel.api.ec2.SecurityGroupSpec
+import com.netflix.spinnaker.keel.api.ec2.old.ClusterV1Spec
+import com.netflix.spinnaker.keel.api.titus.TestContainerVerification
+import com.netflix.spinnaker.keel.api.titus.TitusClusterSpec
+import com.netflix.spinnaker.keel.api.titus.TitusServerGroup.Location
+import com.netflix.spinnaker.keel.core.api.SubmittedDeliveryConfig
+import dev.minutest.junit.JUnit5Minutests
+import dev.minutest.rootContext
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment.NONE
+import strikt.api.expectCatching
+import strikt.assertions.isA
+import strikt.assertions.isEqualTo
+import strikt.assertions.isSuccess
+
+@SpringBootTest(webEnvironment = NONE)
+class DeliveryConfigYamlParsingFilesTests @Autowired constructor(
```

Comment:
I think "files" is unnecessary detail here:
```suggestion
class DeliveryConfigYamlParsingTests @Autowired constructor(
```